crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -97,7 +97,8 @@ module deepbook::clob {
         /// object ID of the `AccountCap` that placed the order
         owner: ID,
         base_asset_quantity_placed: u64,
-        price: u64
+        price: u64,
+        expire_timestamp: u64
```

Comment:
Unfortunately, this is not backward compatible--will have to define a new event type or skip this change.

Reply:
ok, let's revert to maintain backward compatible.

Reply:
I would suggest calling this `OrderPlacedV2`, and moving the old `OrderPlaced` to a clearly marked deprecated section at the bottom of the code.

Reply:
adopt!